### ISSUE TITLE
[TAKE 2 🤦‍♀️ ] ELSEWHERE: Inbound linking from Filecoin docs

### ISSUE THREAD

**Issue by terichadbourne** [web3-storage/docs#78]
## Acceptance Criteria

- [x]  When they hit the specified pages on docs.filecoin.io, users who want to store files (app devs, not miners or aggregators) see a clear indicator above the fold that [web3.storage](http://web3.storage) is the recommended pathway for them and immediately follow a link to web3.storage
- [x]  Callouts to **divert app developers to [Web3.Storage](http://web3.Storage)** while keeping aggregators and miners on the site. We can be somewhat heavy-handed with the messaging here given the specificity to the audience.
    - [x]  P0 Getting started 
    - [x]  [P1 Build landing page](https://docs.filecoin.io/build/)
    - [x]  P1 [Build/getting started page](https://docs.filecoin.io/build/get-started/)
    - [x]  P1 Store landing page
    - [ ]  P2 Docs Landing page - TBC - not a great format for callouts
- [x]  This callout is written in a way that doesn't attract aggregators or miners

- [x]  Extuxry is no longer referred to as "the recommended solution" (please ask Kadir for clarification and review the discussion in [Notion card](https://www.notion.so/protocollabs/Update-docs-filecoin-io-to-push-app-devs-to-web3-storage-ae05189e4fbe4c0f807c8908bdd6cfa2#7525f300d7df4cbf90fbb5d887af2428) before making this edit) in its detailed entries in: 
   - [x] [Store](https://docs.filecoin.io/store/estuary/) 
   - [x] [Build](https://docs.filecoin.io/build/estuary/)
   
   Note: We are NOT making articles similar to those for Web3.Storage, just calling out from the landing pages mentioned above

 **SCOPING NOTE**: For M1 scoping, this task does NOT require a full restructure of the site to better align with personas (app dev, aggregator, miner), just light touchs on the landing page and getting started pages.

- **DO NOT MERGE** until after launch of web3.storage

## Partners in crime

- JV on positioning of intended user and product pitch

## Executive approvals required

- Kadir
- JV

**Comment by terichadbourne** [comment]
@johndmulhausen Can you please point from this issue to your work in progress on the Filecoin docs site?

**Comment by johnnymatthews** [comment]
The only task I haven't completed is:

>  P2 Docs Landing page - TBC - not a great format for callouts

Mainly because the homepage for this site is not a good place to have callouts.

**Comment by terichadbourne** [comment]
Now done in a new PR: https://github.com/filecoin-project/filecoin-docs/pull/986
